conveyal/analysis-ui
# Update user manual and changelog

## Description

Belated updates related to #928 and other changes.

## How to test
1. Check changelog
2. Check read the docs build
